Makefile

```diff
@@ -25,10 +27,18 @@ $(FRONTEND_TARGET): $(FRONTEND_DEPS)
 backend: $(FRONTEND_TARGET)
 	dune build $(FLAGS) @install
 
-run: backend
-	dune exec -- ftw --db=tests/test.db
+stop:
+	killall ftw_backend 2>/dev/null || true
 
-tests: backend
+run: backend stop
+	dune exec -- ftw_backend --db=tests/test.db
+
+frontend_dev: $(FRONTEND_DEPS) stop
+	dune build $(FLAGS) @install
```

Comment:
why not depend on `backend` ? at worst it will compile the frontend once and then not use it